content/en/docs/concepts/workloads/pods/pod-overview.md

```diff
@@ -74,9 +74,9 @@ Here are some examples of workload resources that manage one or more Pods:
 
 ## Pod templates
 
-Generally: controllers for workload resources use a Pod template (that you provide) to create the Pods for that workload. The controller is responsible for managing those Pods on your behalf.
+Generally: controllers for {{< glossary_tooltip text="workload" term_id="workload" >}} resources use a Pod template (that you provide) to create the Pods for that workload. The controller is responsible for managing those Pods on your behalf.
 
-PodTemplates are specifications for creating pods, and are included in other objects such as
+PodTemplates are specifications for creating pods, and are included in other workload resources such as
```

Comment:
line 79, nit: capitalize Pods?

Reply:
Text styling aside, https://kubernetes.io/blog/2015/06/the-distributed-system-toolkit-patterns/ is (still) a good article and I'm happy to link to it.

I'll drop the link to https://kubernetes.io/blog/2016/06/container-design-patterns/